clientv3/snapshot/v3_snapshot.go

```diff
@@ -166,6 +167,16 @@ func (s *v3Manager) Status(dbPath string) (ds Status, err error) {
 	h := crc32.New(crc32.MakeTable(crc32.Castagnoli))
 
 	if err = db.View(func(tx *bolt.Tx) error {
+		// check database integrity first
+		var count int
+		var db_err_strings []string
+		for db_err := range tx.Check() {
+			db_err_strings = append(db_err_strings, db_err.Error())
+			count++
+		}
+		if count > 0 {
+			return fmt.Errorf("Database corrupted! %d errors found.\n"+strings.Join(db_err_strings, "\n"), count)
```

Comment:
database corrupted. 

Error message should not start with capital letter.

Reply:
Thanks!